.gitignore

```diff
@@ -36,3 +36,4 @@ conda.tmp/
 *.iml
 *.ipr
 *.iws
+dev/docker.bashrc
```

Comment:
What's the use case for this file?

Reply:
This file is now checked in the repo because we need it on other places too.